pkg/alexa/l10n/l10n.go

```diff
@@ -0,0 +1,246 @@
+package l10n
+
+import (
+	"fmt"
+	"github.com/drpsychick/alexa-go-cloudformation-demo/pkg/alexa"
+	"math/rand"
+	"time"
+)
+
+// Default keys
+const (
```

Comment:
These seems to be skill specific, why not put them in `gen` rather then in `l10n`?

Reply:
Initially it came from `l10n` because there was no `gen`. Now that I think about it, yes it could make sense in `gen`, but when you define `Locale` for me it seems weird to use `l10n` and `gen`
```
**l10n**.Locale{
  TextSnippets: map[string][]string{
    **gen**.KeySkillName: []string{"my name"},
  }
}
```

Now it's a bit funky, but still consistent: you can use `l10n` without `gen`, but not the other way around - and that makes sense.
* `l10n` defines the interfaces used in `gen`
* `l10n` defines the default keys used in `gen`